test/functionalTest/cases/2871_fwd_update_ok_but_not_found_response/fwd_update_ok_but_not_found_response.test

```diff
@@ -78,28 +79,58 @@ echo
 echo "02. Create entity directly in DB"
 echo "================================"
 doc='{
-  "_id":ObjectId("5b1ff2ee49597334c9b5efc7"),
-  "expiration":NumberLong(1531412462),
-  "servicePath":"/ss1",
-  "format":"JSON",
-  "contextRegistration":[
-    {
-      "entities":[
-        {
-          "id":"entity1",
-          "type":"device"
-        }
-      ],
-      "attrs":[
-        {
-          "name":"turn",
-          "type":"string",
-          "isDomain":"false"
-        }
-      ],
-      "providingApplication":"http://localhost:9997/bug2871"
-    }
-  ]
+        "_id": {
```

Comment:
Its funny... this was completely wrong (a reg doc inserted in the entities collection) but the .test was working anyway. I have recovered the entity example in the #2871 issue and check that the .test is failing with versions of CB previous to the fix.

NTC (just informative)